smart-contracts/.env.example

```diff
@@ -33,3 +33,5 @@ MAINNET_GAS_PRICE=10000000
 # If you are deploying a new instance of peggy that needs to use eRowan,
 # set this variable to the address of the eRowan smart contract
 EROWAN_ADDRESS='0x0d8cc4b8d15D4c3eF1d70af0071376fb26B5669b'
+# Replcace example ALCHEMY_URL with your Alchemy's project id
+ALCHEMY_URL=https://eth-mainnet.alchemyapi.io/v2/x4YoKdIFoqysvsttnz81U4JHqlVPxsAH
```

Comment:
Let's remove this secret.
We should also change the comment to say `Replace with your own web3 provider url`